### PR TITLE
Notes for npm scripts

### PR DESCRIPTION
**Pull request by farhatcode**
Adding a note for enumerate-phrases npm script.

### CODE REVIEW

File: docs/writing-an-intent.md [hunk 1]
```diff
@@ -178,3 +178,15 @@ Use `log.info()` (debug, etc) for any logging you plan to leave in the code. `lo
...
+
+## NPM scripts
+
+NPM scripts are simply terminal commands. The npm scripts which are available to use are listed in package.json file. `enumerate-phrases` is a project specific/custom npm script.
+
+**enumerate-phrases:** lists all the matched phrases for the specified intent using the auto-generated metadata for finding matches. This metadata is generated from all the .toml files containing intents description. 
+
+You can execute this script as:
+```js 
+npm run enumerate-phrases --someIntent.command
```
Comment (ianb) [inline docs/writing-an-intent.md]: There needs to be a space between `--` and `someIntent.command`. This is something `npm run` does – it grabs all the options for itself up until `--` and then the rest of the commands go to the script. The script here looks for a pattern.

File: docs/writing-an-intent.md [hunk 2]
```diff
@@ -178,3 +178,15 @@ Use `log.info()` (debug, etc) for any logging you plan to leave in the code. `lo
...
+```
+For example, Running the command `npm run enumerate-phrases --navigation.bangSearch` lists all the matched phrases for navigation.bangSearch intent.
```
Comment (ianb) [inline docs/writing-an-intent.md]: Same here – the command you give returns every possible phrase (all 76k of them!)

File: docs/writing-an-intent.md (hunk 1 above)
Comment (farhatcode) [reply docs/writing-an-intent.md]: I have made correction to the command.

**Comment by ianb** [comment]
Looks good, thanks!

**Pull request merged**